weasel/errors.py

```diff
@@ -8,17 +8,23 @@ def __getattribute__(self, code):
 
 
 class Warnings(metaclass=ErrorsWithCodes):
-    W091 = "Could not clean/remove the temp directory at {dir}: {msg}."
+    # File system
+    W201 = "Could not clean/remove the temp directory at {dir}: {msg}."
 
 
 class Errors(metaclass=ErrorsWithCodes):
-    E095 = (
+    # API - Datastructure
+    E001 = (
         "Can't write to frozen dictionary. This is likely an internal "
         "error. Are you writing to a default function argument?"
     )
-    E852 = "The tar file pulled from the remote attempted an unsafe path " "traversal."
-    E927 = (
+    E002 = (
         "Can't write to frozen list. Maybe you're trying to modify a computed "
         "property or default function argument?"
     )
-    E970 = "Can not execute command '{str_command}'. Do you have '{tool}' installed?"
+
+    # Workflow
+    E101 = "Can not execute command '{str_command}'. Do you have '{tool}' installed?"
```

Comment:
What if we run into more than a hundred errors per "category"?

Reply:
Ok I've spaced the codes out, to respect what seems to be an unspoken error code rule: https://beta.ruff.rs/docs/rules